server/app/assets/javascripts/admin_programs.ts

```diff
@@ -96,6 +96,7 @@ class AdminPrograms {
 
   static async copyProgramLinkToClipboard(programLink: string) {
     const succeeded = await AdminPrograms.tryCopyToClipboard(programLink)
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const toastController = window['toastController'] as any
```

Comment:
Can we import ToastController and do window['toastController'] as ToastController?

Reply:
That is a good suggestion for a future PR